src/apps/components/pages/CultureDetailsTab.svelte

```diff
@@ -1,168 +1,173 @@
 <svelte:options accessors={true} />
 
 <script>
-  import { getContext } from "svelte";
-  import { localize } from "@typhonjs-fvtt/runtime/svelte/helper";
+    import { getContext } from "svelte";
+    import { localize } from "@typhonjs-fvtt/runtime/svelte/helper";
 
-  import DropArea from "../DropArea.svelte";
+    import DropArea from "../DropArea.svelte";
 
-  import LinkedDocumentsHelper from "../../utils/LinkedDocumentsHelper";
-  import updateDocumentDataFromField from "../../utils/updateDocumentDataFromField";
-  import localizeLanguage from "../../utils/localizeLanguage";
+    import LinkedDocumentsHelper from "../../utils/LinkedDocumentsHelper";
+    import updateDocumentDataFromField from "../../utils/updateDocumentDataFromField";
+    import prepareLanguageProficiencies from "../../handlers/prepareLanguageProficiencies";
 
-  import LanguageSelectDialog from "../../dialogs/initializers/LanguageSelectDialog";
+    import LanguageSelectDialog from "../../dialogs/initializers/LanguageSelectDialog";
 
-  const item = getContext("item");
+    const item = getContext("item");
 
-  const linkedFeatures = new LinkedDocumentsHelper($item, "features", {
-    validate: (obj) =>
-      obj.type === "feature" && obj.system.featureType === "culture",
-  });
-
-  function handleAddFeature(event) {
-    const [dragEvent, _] = event.detail;
+    const linkedFeatures = new LinkedDocumentsHelper($item, "features", {
+        validate: (obj) =>
+            obj.type === "feature" && obj.system.featureType === "culture",
+    });
 
-    try {
-      const { uuid } = JSON.parse(dragEvent.dataTransfer.getData("text/plain"));
-      linkedFeatures.add({ uuid, optional: false });
-    } catch (err) {
-      console.error(err);
+    function handleAddFeature(event) {
```

Comment:
If we expect to use these handlers in other item types are well, we could probably turn these into separate files and import them